x/distribution/client/cli/query.go

```diff
@@ -54,24 +57,26 @@ func GetCmdQueryParams(queryRoute string, cdc *codec.Codec) *cobra.Command {
 			}
 
 			var params types.Params
-			if err := cdc.UnmarshalJSON(res, &params); err != nil {
+			if err := clientCtx.JSONMarshaler.UnmarshalJSON(res, &params); err != nil {
 				return fmt.Errorf("failed to unmarshal params: %w", err)
 			}
 
 			return clientCtx.PrintOutput(params)
 		},
 	}
+
+	return flags.GetCommands(cmd)[0]
```

Comment:
Is this redundant with: https://github.com/cosmos/cosmos-sdk/blob/203de6563d2244af06d7273086d9f56dddb6dbdd/x/distribution/client/cli/query.go#L28-L35?

Reply:
Yes, good catch actually!